db/migrate/20221104075655_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+
+      t.timestamps
+    end
```

Comment:
Not a big deal at this point, as this migration contains only the timestamps for users, we might consider merging it into the next migration. Up to you Sergiu.